### ISSUE TITLE
Add BED files loading

### ISSUE THREAD

**Issue by asl** [asl/BandageNG#22]
This could be used to apply any kind of annotation to the graph.

We need to support the basic columns:
  - chrom
  - chromStart
  - chromEnd

As well as:
  - name
  - strand
  - rgb

The formal specification is at https://samtools.github.io/hts-specs/BEDv1.pdf

**Comment by ekg** [comment]
We've been testing this in gfaestus and it's proved very very useful.
Strong support!!

**Comment by wafemand** [comment]
I managed to parse and visualize bed files with all standard fields

E.g.:
```
test.bed
709047	10000	40000	kek	10	+	255,0,0	2	15000	30000	5000,3000	0,37000
```
Annotation of the node with length 45541 nucls looks like this:
![image](https://user-images.githubusercontent.com/28672835/162036129-1aa15c6e-1ff3-4cdc-85c0-997557ca0330.png)

Now I'm working on UI and internal representation.

**Comment by asl** [comment]
Fixed by #32

**Comment by AndreaGuarracino** [comment]
Did #32 solve the problem? Usually, in the BED file, the first column specifies the name of the paths, not the segment names in the GFA.
